liquibase-standard/src/test/groovy/liquibase/parser/core/yaml/YamlChangeLogParser_RealFile_Test.groovy

```diff
@@ -651,4 +651,18 @@ public class YamlChangeLogParser_RealFile_Test extends Specification {
         cleanup:
         Scope.getCurrentScope().getSingleton(ChangeFactory.class).unregister("createTableExample");
     }
+
+
+    def "change set have to empty when change log has only databaseChangeLog tag"() throws ChangeLogParseException {
```

Comment:
This test name is not as clear as it could be. I suggest changing this to "Verify Liquibase returns zero changesets when a YAML changelog only has the databaseChangeLog tag."

Should we add a test for JSON as well? 

CC @filipelautert @rberezen